BillTrack.Application/BillTrack.Application.csproj

```diff
@@ -11,6 +11,7 @@
     </ItemGroup>
 
     <ItemGroup>
+      <PackageReference Include="AWSSDK.S3" Version="3.7.403.5" />
```

Comment:
AWS SDK has tons of versions. And its parts could be different versions. Only the same versions are tested for compatibility.
If there is no requirement about a version number there is a point in using the same version number for all AWS SDK modules.